frontend/projects/upgrade/src/app/core/error-event-bus/store/error-event-bus.model.ts

```diff
@@ -0,0 +1,7 @@
+import { SERVER_ERROR } from 'upgrade_types';
+
+export interface NewErrorEvent {
+  type: SERVER_ERROR | string;
```

Comment:
Do you need `string` here? Because `NewErrorEvent` is getting created from `errorTypeToActionsMap` where you will add only `SERVER_ERROR`.

Reply:
probably not, I'm not sure what my thought was there.